src/cljs/rems/catalogue.cljs

```diff
@@ -124,10 +108,17 @@
 
 (defn catalogue-page []
   (let [language @(rf/subscribe [:language])
-        catalogue @(rf/subscribe [::catalogue])
         loading-catalogue? @(rf/subscribe [::loading-catalogue?])
         drafts @(rf/subscribe [::draft-applications])
-        loading-drafts? @(rf/subscribe [::loading-drafts?])]
+        loading-drafts? @(rf/subscribe [::loading-drafts?])
+        table-spec {:id :catalogue
```

Comment:
spec is very much overloaded with Clojure spec so I'd use definition as earlier or something else entirely.